### ISSUE TITLE
ErrorController::noRouteFound

### ISSUE THREAD

**Issue by magicsunday** [fisharebest/webtrees#1622]
After login as admin the ErrorController::noRouteFound is triggered and redirects (line 57) me to "admin_trees_manage.php" without any error message.

**Comment by fisharebest** [comment]
Were you importing a GEDCOM file?
Are all your trees private?

If there are no available trees, webtrees will redirect you to admin_trees_manage.php.
But if you are not an administrator, this route does not exist.

I think this might be a valid bug, so I will re-open it.

**Comment by magicsunday** [comment]
I did no import. I just copied my whole database to a new dev environment and updated to the latest version from master-branch. My tree is private. I'm the administrator and only user of the website. 

For some unknown reason the redirect vanished but now its happening again. 

- Login as administrator
- Only one private tree in database
- Expected result: Show dashboard, not admin tree manage page as before
